mindsdb/api/http/initialize.py

```diff
@@ -16,8 +24,87 @@ class Swagger_Api(Api):
     def specs_url(self):
         return url_for(self.endpoint("specs"), _external=False)
 
+
+def initialize_static():
+    this_file_path = os.path.abspath(inspect.getfile(inspect.currentframe()))
+    static_path = Path(this_file_path).parent.joinpath('static/')
+    static_path.mkdir(parents=True, exist_ok=True)
+
+    try:
+        res = requests.get('https://raw.githubusercontent.com/mindsdb/mindsdb_gui_web/master/compatible-config.json?token=AA7S27R5CPBEUKNEONQJNBC7LPBJK')
+    except ConnectionError as e:
+        print(f'Is no connection. {e}')
+        return False
+
+    versions = res.json()
+
+    current_mindsdb_lv = LooseVersion(mindsdb_version)
+
+    gui_versions = {}
+    gui_version_lv = None
+    max_mindsdb_lv = None
+    for el in versions['mindsdb']:
+        mindsdb_lv = LooseVersion(el['mindsdb_version'])
+        gui_lv = LooseVersion(el['gui_version'])
+        if mindsdb_lv.vstring not in gui_versions or gui_lv > gui_versions[mindsdb_lv.vstring]:
+            gui_versions[mindsdb_lv.vstring] = gui_lv
+        if max_mindsdb_lv is None or max_mindsdb_lv < mindsdb_lv:
+            max_mindsdb_lv = mindsdb_lv
+    if current_mindsdb_lv.vstring in gui_versions:
+        gui_version_lv = gui_versions[current_mindsdb_lv.vstring]
+    else:
+        gui_version_lv = gui_versions[max_mindsdb_lv.vstring]
+
+    current_gui_version = None
+
+    version_txt_path = static_path.joinpath('version.txt')
+    if version_txt_path.is_file():
+        with open(version_txt_path, 'rt') as f:
+            current_gui_version = f.readline()
+    if current_gui_version is not None:
+        current_gui_lv = LooseVersion(current_gui_version)
+        if current_gui_lv >= gui_version_lv:
+            return True
+    print('New version of GUI available. Downloading...')
+
+    shutil.rmtree(static_path)
+    static_path.mkdir(parents=True, exist_ok=True)
+
+    ACCESS_KEY = ""
+    SECRET_KEY = ""
+
+    try:
+        s3 = boto3.client('s3', aws_access_key_id=ACCESS_KEY, aws_secret_access_key=SECRET_KEY)
+        s3.download_file('mindsdb-web-builds', 'index.html', str(static_path.joinpath('index.html')))
+        css_zip_path = str(static_path.joinpath('css.zip'))
+        js_zip_path = str(static_path.joinpath('js.zip'))
+        s3.download_file('mindsdb-web-builds', 'css-V' + gui_version_lv.vstring + '.zip', css_zip_path)
+        s3.download_file('mindsdb-web-builds', 'js-V' + gui_version_lv.vstring + '.zip', js_zip_path)
```

Comment:
Don't include boto3 for the sake of downloading some files, since these will be public anyway.

It's an extra dependency that is to be avoided, but specifically, in the case of boto3, it's an extra dependency that can interact badly with a lot of other things the user might have in his env since it's sometimes problematic to installed and required by many packages.